PLATFORM_SUPPORT.md

```diff
@@ -31,12 +31,12 @@ Community members may provide assistance with these platforms.
 
 ## Platforms
 
-|Architecture|Ubuntu Jammy (22.04)|Ubuntu Focal (20.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Debian Buster (10)|Android 12 | QNX 7.1   |
-|------------|--------------------|--------------------|--------------------|-------------------|------------------|-----------|-----------|
-|amd64       |Tier 1 [^a][^s]     |Tier 3 [^a][^s]     |Tier 1 [^s]         |Tier 1 [^a][^s]    |Tier 3 [^s]       |Tier 3 [^s]|Tier 3 [^s]|
-|amd32       |                    |                    |                    |Tier 2 [^a][^s]    |                  |           |           |
-|arm64       |Tier 1 [^a][^s]     |Tier 3 [^a][^s]     |                    |                   |Tier 3 [^s]       |Tier 3 [^s]|Tier 3 [^s]|
-|arm32       |Tier 3 [^s]         |Tier 3 [^s]         |                    |                   |Tier 3 [^s]       |Tier 3 [^s]|           |
+|Architecture|Ubuntu Jammy (22.04)|Ubuntu Focal (20.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Debian Buster (10)|Android 12 |Android 13 | QNX 7.1   |
```

Comment:
I think we could add Windows 11 (VS2022) as Tier 3. It is what I am using on my brand-new laptop.